superset-frontend/src/components/Icons/index.tsx

```diff
@@ -0,0 +1,175 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import AntdEnhancedIcons from './AntdEnhanced';
+import { Icon } from './Icon';
+import IconType from './IconType';
+
+export default {
+  ...AntdEnhancedIcons,
+  Alert: (props: IconType) => <Icon path="alert" {...props} />,
```

Comment:
Not sure if this makes sense, but maybe worth exploring, to shrink the file some... if they all fit the same pattern of naming conventions and such.

```js
// import lodash at the top...

const iconPaths = [alert_solid, alert_solid_small]; // etc... this is the list we would need to maintain.
const iconOverrides = {};
iconPaths.forEach(path => {
 const keyName = _.startCase('path').replace(/ /g,''); // lodash strips dashes and capitalizes, then we remove spaces.
 iconOverrides[keyName] = (props: IconType) => <Icon path={path} {...props} />;
});

export default {
  ...AntdEnhancedIcons,
  ...iconOverrides,
}

```

Reply:
That's a clever idea!!